About portal, Support, Rules

sivxkxn opened the pull request:
#1292

litvinets reviewed (changes requested):
can you please fix a bit of styling? It should be the same as before 
![image](https://user-images.githubusercontent.com/27493959/177368269-297d532c-6294-4a31-b08a-e5f09d1d1efb.png)
![image](https://user-images.githubusercontent.com/27493959/177368305-6f9fc797-89dd-42fb-8dad-d6a8c8e6b359.png)

litvinets reviewed (changes requested):
Please, can you additionally add routing back on successful update 
Currently the routing is broken 
![image](https://user-images.githubusercontent.com/27493959/177369837-3a31d0ec-e1f1-4822-ae8a-0644eb304de8.png)

Err: polyfills.07a5c2670fe49c10aa0d.js:1 Unhandled Promise rejection: Cannot match any routes. URL Segment: 'admin-tools/platform/AboutPortal' ; Zone: <root> ; Task: Promise.then ; Value: Error: Cannot match any routes. URL Segment: 'admin-tools/platform/AboutPortal'

sivxkxn commented:
@litvinets 
1. replaced services with store;
2. added routing after Save changes;
3. left the styles as they were, the font of the text depends on the scale in the browser;
4. fixed build ;

litvinets commented on src/app/shell/admin-tools/platform/platform-info/info-edit/info-edit.component.html:
```diff
@@ -29,8 +29,8 @@ <h3 class="wrapper-title">РЕДАГУВАННЯ {{editTitle}}</h3>
     <div fxLayout="row" fxLayoutAlign="center center" class="footer">
       <button mat-raised-button class="btn btn-cancel" (click)="onBack()">Скасувати</button>
-      <button class="btn" mat-button type="submit"[disabled]="PlatformInfoItemArray.invalid || titleFormControl.invalid" (click)="onSubmit()">Зберегти</button>
+      <a [routerLink]="setRoute()"><button class="btn" mat-button type="submit"[disabled]="PlatformInfoItemArray.invalid || titleFormControl.invalid"  (click)="onSubmit()">Зберегти</button></a>
```
this.router.navigate([`/admin-tools/platform/`, { queryParams: {page: payload}}]); 
![image](https://user-images.githubusercontent.com/27493959/177571102-4d413a71-1c51-41f2-bfc7-6a5efe36c789.png)

sivxkxn replied:
done, but it is not working as expected

litvinets commented on src/app/shared/store/admin.state.ts:
```diff
@@ -204,7 +205,7 @@ export class AdminState {
       new ShowMessageBar({ message: 'Інформація про портал успішно відредагована', type: 'success' })
     ]);
-    this.router.navigate([`/admin-tools/platform/${payload}`]);
+    this.router.navigate([`/admin-tools/platform/`, { queryParams: {page: payload}}]);
```
instaed of payload pass type
